src/main/scala/org/constellation/infrastructure/endpoints/SnapshotEndpoints.scala

```diff
@@ -173,6 +174,14 @@ class SnapshotEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F] {
         .map(_.asJson)
         .flatMap(Ok(_))
   }
+
+  private def getTotalSupply(snapshotService: SnapshotService[F]): HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "total-supply" =>
```

Comment:
There are some endpoints written in camel case like `latestMajorityHeight`. I like kebab case more for URLs, but I'm afraid that it could introduce inconsistency in our API (unless there is already inconsistency, then that's fine let's continue with kebab case).

Reply:
"unless there is already inconsistency" - there is :D no worries :D

Reply:
good catch. we have both in the codebase and I prefer kebab-case more as well.
@kpudlik @TheMMaciek ?

Reply:
There are both styles already in the codebase :D.

Reply:
So I guess we can go with kebab case from now on.